src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.ts

```diff
@@ -39,7 +43,13 @@ export class ModelingExamSubmissionComponent extends ExamSubmissionComponent imp
         return this.exercise;
     }
 
-    onActivate(): void {}
+    onActivate(): void {
+        this.cdr.reattach();
+    }
+
+    onDeactivate(): void {
+        this.cdr.reattach();
```

Comment:
should this not be `detach()` according to https://angular.io/api/core/ChangeDetectorRef

Reply:
See 25ab3af2b18efb7ede588c74cbb4ea22919ded30